### PR TITLE
tests: update centos images and add new centos 9 image

### PR DESCRIPTION
**Pull request by sergiocazzolato**
This change includes new image for centos 9.

Centos 9 it going to be enabled in a following pr with some other fixes.

Also centos 8 image is updated to use the once from snapd spread project

### CODE REVIEW

**Review by mardy — COMMENTED** [review]
LGTM, but the centos8 tests fail in the prepare stage. I see that you changed the image for centos-8; please explain this change in the commit message, to make sure it's intended.

**Comment by sergiocazzolato** [comment]
@mardy hi, I just updated the centos 8 image in google, the problem is that the google SDK repo which is coming with the image fails to update.

File: spread.yaml [hunk 1]
```diff
@@ -164,7 +164,13 @@ backends:
             - centos-8-64:
                   workers: 6
                   storage: preserve-size
-                  image: centos-stream-8
+                  image: centos-8-64
```
Comment (bboozzoo) [inline spread.yaml:167]: Those are our images based on centos-stream-8 from GCE?

File: spread.yaml [hunk 2]
```diff
@@ -164,7 +164,13 @@ backends:
...
+            - centos-9-64:
+                  workers: 6
+                  storage: preserve-size
+                  image: centos-9-64
+                  manual: true
```
Comment (bboozzoo) [inline spread.yaml:172]: Do we need to keep this manual?

File: spread.yaml (hunk 1 above)
Comment (sergiocazzolato) [reply spread.yaml:167]: centos-stream-8 is the upstream image, that one is broken and it has not the test dependencies installed
centos-8-64 is our image which has the test dependencies installed

File: spread.yaml (hunk 2 above)
Comment (sergiocazzolato) [reply spread.yaml:172]: I'll remove the manual as it is not included in the workflow, so far it is failing to run
I'll add a following pr to enable centos-9, but for that we need to have the xdelta3 dependency available

**Review by mvo5 — APPROVED** [review]
Thanks, spread changes look good

**Pull request merged**